src/website/src/app/documentation/demos/accordion/accordion.demo.html

```diff
@@ -0,0 +1,38 @@
+<clr-doc-wrapper [ng]="ng" [ui]="ui" [title]="title" [newLayout]="newLayout">
+  <article>
+    <h5 class="component-summary" id="stepper-header">
+      A accordion allows generic content to be collapsed and allow users to expand to show more detail.
+    </h5>
+
+    <div id="design-guidelines">
+      <h3>Design Guidelines</h3>
+
+      <h3>Usage</h3>
+      <p>
+        Accordions are designed for use in the main content area. Use a accordion when you want to progressively
+        disclose data. This is useful when the set of information is large or you want to enable users to reveal more
+        complex or less frequently used data as necessary.
+      </p>
+    </div>
+
+    <div id="code-examples">
+      <h3 id="examples">Code &amp; Examples</h3>
+
+      <p>
+        Accordions by default only allow one panel to be open at a time. To enable multiple panels set
+        <code>clrAccordionMultiPanel="true"</code>
+        on the `clr-accordion` component.
+      </p>
+
+      <p>
+        Leverage our <code>*clrIfExpanded</code> structural directive on the <code>clr-accordion-panel</code> to only
```

Comment:
This is the last comment before the example, but you are not showing it in the example code. It's enough for me or you, but this may happen to be the first such component of the user, so an example will be nice. Not sure if we need 2 separate examples or just one will be enough.

Reply:
Hmm I'm thinking maybe one and we default to users using the `*clrIfExpand`?

Reply:
Updated as default example.